exporter/prometheus/prometheus.go

```diff
@@ -47,6 +47,7 @@ type Options struct {
 	Namespace string
 	Registry  *prometheus.Registry
 	OnError   func(err error)
+	Labels    prometheus.Labels
```

Comment:
I think we should call this "ConstLabels" (to distinguish them from tag-derived labels) and document that these will be set on all views.
```suggestion
	ConstLabels    prometheus.Labels // ConstLabels will be set as labels on all views.
```

Reply:
Done! Couldn't accept the suggestion because it wanted to make a new commit and it'd only fix part of it :(.